recipes/pybufr-ecmwf/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]; then
+  export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib
```

Comment:
This is sightly different from what @patricksnape  said in https://github.com/conda-forge/staged-recipes/pull/1189#issuecomment-237926406 but it works to find the shared library. However, now we have a new error on `OS X` that I have no idea how to fix.

```shell
gfortran:f77: f2py_build/src.macosx-10.6-x86_64-2.7/f2py_build/ecmwfbufr-f2pywrappers.f
/Users/travis/miniconda/envs/_build/bin/gfortran -Wall -g -arch x86_64 f2py_build/f2py_build/src.macosx-10.6-x86_64-2.7/f2py_build/ecmwfbufrmodule.o f2py_build/f2py_build/src.macosx-10.6-x86_64-2.7/fortranobject.o f2py_build/f2py_build/src.macosx-10.6-x86_64-2.7/f2py_build/ecmwfbufr-f2pywrappers.o -L./ -L/Users/travis/miniconda/envs/_build/lib/gcc/x86_64-apple-darwin11.4.2/4.8.5 -L/Users/travis/miniconda/envs/_build/lib -lbufr -lgfortran -o ./ecmwfbufr.so
Undefined symbols for architecture x86_64:
  "_PyArg_ParseTupleAndKeywords", referenced from:
      _f2py_rout_ecmwfbufr_retrieve_settings in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_close_fortran_stdout in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_big_endian in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs2 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs1 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs0 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buukey in ecmwfbufrmodule.o
      ...
  "_PyCObject_AsVoidPtr", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _F2PyCapsule_AsVoidPtr in fortranobject.o
  "_PyCObject_FromVoidPtr", referenced from:
      _fortran_getattr in fortranobject.o
      _F2PyCapsule_FromVoidPtr in fortranobject.o
  "_PyCObject_Type", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _F2PyCapsule_Check in fortranobject.o
  "_PyComplex_Type", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _int_from_pyobj in ecmwfbufrmodule.o
      _double_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbseek in ecmwfbufrmodule.o
      ...
  "_PyDict_DelItemString", referenced from:
      _fortran_setattr in fortranobject.o
  "_PyDict_GetItemString", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _fortran_getattr in fortranobject.o
  "_PyDict_New", referenced from:
      _PyFortranObject_New in fortranobject.o
      _PyFortranObject_NewAsAttr in fortranobject.o
      _fortran_setattr in fortranobject.o
  "_PyDict_SetItemString", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _fortran_getattr in fortranobject.o
      _F2PyDict_SetItemString in fortranobject.o
      _PyFortranObject_New in fortranobject.o
      _fortran_setattr in fortranobject.o
  "_PyErr_Clear", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _int_from_pyobj in ecmwfbufrmodule.o
      _double_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbseek in ecmwfbufrmodule.o
      ...
  "_PyErr_Format", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _fortran_call in fortranobject.o
  "_PyErr_NewException", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
  "_PyErr_NoMemory", referenced from:
      _fortran_getattr in fortranobject.o
  "_PyErr_Occurred", referenced from:
      _f2py_rout_ecmwfbufr_retrieve_settings in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_close_fortran_stdout in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_big_endian in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs2 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs1 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs0 in ecmwfbufrmodule.o
      _long_long_from_pyobj in ecmwfbufrmodule.o
      ...
  "_PyErr_Print", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _F2PyDict_SetItemString in fortranobject.o
  "_PyErr_SetString", referenced from:
      _f2py_rout_ecmwfbufr_buprs2 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs1 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs0 in ecmwfbufrmodule.o
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _int_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buukey in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_bupkey in ecmwfbufrmodule.o
      ...
  "_PyExc_AttributeError", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _fortran_setattr in fortranobject.o
  "_PyExc_ImportError", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
  "_PyExc_MemoryError", referenced from:
      _string_from_pyobj.constprop.3 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_getcode in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_open_fortran_stdout in ecmwfbufrmodule.o
  "_PyExc_RuntimeError", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _fortran_call in fortranobject.o
  "_PyExc_TypeError", referenced from:
      _fortran_call in fortranobject.o
      _array_from_pyobj in fortranobject.o
  "_PyExc_ValueError", referenced from:
      _string_from_pyobj.constprop.3 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_getcode in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_open_fortran_stdout in ecmwfbufrmodule.o
      _array_from_pyobj in fortranobject.o
  "_PyFloat_Type", referenced from:
      _double_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_fmmh in ecmwfbufrmodule.o
  "_PyImport_ImportModule", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
  "_PyLong_AsLongLong", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbseek in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbwrite in ecmwfbufrmodule.o
  "_PyMem_Free", referenced from:
      _fortran_dealloc in fortranobject.o
      _fortran_getattr in fortranobject.o
  "_PyMem_Malloc", referenced from:
      _fortran_getattr in fortranobject.o
  "_PyModule_GetDict", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
  "_PyNumber_Float", referenced from:
      _double_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_fmmh in ecmwfbufrmodule.o
  "_PyNumber_Int", referenced from:
      _int_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buepwtc in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_get_free_unit in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_retrieve_jbpw in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_bunexs in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_get_tables in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_bustop in ecmwfbufrmodule.o
      ...
  "_PyNumber_Long", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbseek in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbwrite in ecmwfbufrmodule.o
  "_PyOS_snprintf", referenced from:
      _fortran_getattr in fortranobject.o
  "_PyObject_GetAttrString", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _int_from_pyobj in ecmwfbufrmodule.o
      _double_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbseek in ecmwfbufrmodule.o
      ...
  "_PyObject_IsTrue", referenced from:
      _f2py_rout_ecmwfbufr_mbufr_mars_filter in ecmwfbufrmodule.o
  "_PyObject_SetAttrString", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
  "_PyObject_Str", referenced from:
      _string_from_pyobj.constprop.3 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_getcode in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_open_fortran_stdout in ecmwfbufrmodule.o
  "_PySequence_Check", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _int_from_pyobj in ecmwfbufrmodule.o
      _double_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbseek in ecmwfbufrmodule.o
      ...
  "_PySequence_GetItem", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _int_from_pyobj in ecmwfbufrmodule.o
      _double_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbseek in ecmwfbufrmodule.o
      ...
  "_PyString_AsString", referenced from:
      _fortran_repr in fortranobject.o
  "_PyString_ConcatAndDel", referenced from:
      _fortran_getattr in fortranobject.o
  "_PyString_FromFormat", referenced from:
      _fortran_repr in fortranobject.o
  "_PyString_FromString", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _fortran_repr in fortranobject.o
      _fortran_getattr in fortranobject.o
  "_PyString_FromStringAndSize", referenced from:
      _fortran_getattr in fortranobject.o
  "_PyType_IsSubtype", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _int_from_pyobj in ecmwfbufrmodule.o
      _double_from_pyobj in ecmwfbufrmodule.o
      _string_from_pyobj.constprop.3 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      ...
  "_PyType_Type", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
  "_Py_BuildValue", referenced from:
      _f2py_rout_ecmwfbufr_retrieve_settings in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_close_fortran_stdout in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_big_endian in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs2 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs1 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs0 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buukey in ecmwfbufrmodule.o
      ...
  "_Py_FindMethod", referenced from:
      _fortran_getattr in fortranobject.o
  "_Py_InitModule4_64", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
  "__PyObject_New", referenced from:
      _PyFortranObject_New in fortranobject.o
      _PyFortranObject_NewAsAttr in fortranobject.o
  "__Py_NoneStruct", referenced from:
      _f2py_rout_ecmwfbufr_buprs2 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs1 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs0 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buukey in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_bupkey in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_bucrekey in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_bubox in ecmwfbufrmodule.o
      ...
  "_main", referenced from:
      start in crt1.10.6.o
ld: symbol(s) not found for architecture x86_64
collect2: error: ld returned 1 exit status
Undefined symbols for architecture x86_64:
  "_PyArg_ParseTupleAndKeywords", referenced from:
      _f2py_rout_ecmwfbufr_retrieve_settings in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_close_fortran_stdout in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_big_endian in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs2 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs1 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs0 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buukey in ecmwfbufrmodule.o
      ...
  "_PyCObject_AsVoidPtr", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _F2PyCapsule_AsVoidPtr in fortranobject.o
  "_PyCObject_FromVoidPtr", referenced from:
      _fortran_getattr in fortranobject.o
      _F2PyCapsule_FromVoidPtr in fortranobject.o
  "_PyCObject_Type", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _F2PyCapsule_Check in fortranobject.o
  "_PyComplex_Type", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _int_from_pyobj in ecmwfbufrmodule.o
      _double_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbseek in ecmwfbufrmodule.o
      ...
  "_PyDict_DelItemString", referenced from:
      _fortran_setattr in fortranobject.o
  "_PyDict_GetItemString", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _fortran_getattr in fortranobject.o
  "_PyDict_New", referenced from:
      _PyFortranObject_New in fortranobject.o
      _PyFortranObject_NewAsAttr in fortranobject.o
      _fortran_setattr in fortranobject.o
  "_PyDict_SetItemString", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _fortran_getattr in fortranobject.o
      _F2PyDict_SetItemString in fortranobject.o
      _PyFortranObject_New in fortranobject.o
      _fortran_setattr in fortranobject.o
  "_PyErr_Clear", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _int_from_pyobj in ecmwfbufrmodule.o
      _double_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbseek in ecmwfbufrmodule.o
      ...
  "_PyErr_Format", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _fortran_call in fortranobject.o
  "_PyErr_NewException", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
  "_PyErr_NoMemory", referenced from:
      _fortran_getattr in fortranobject.o
  "_PyErr_Occurred", referenced from:
      _f2py_rout_ecmwfbufr_retrieve_settings in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_close_fortran_stdout in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_big_endian in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs2 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs1 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs0 in ecmwfbufrmodule.o
      _long_long_from_pyobj in ecmwfbufrmodule.o
      ...
  "_PyErr_Print", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _F2PyDict_SetItemString in fortranobject.o
  "_PyErr_SetString", referenced from:
      _f2py_rout_ecmwfbufr_buprs2 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs1 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs0 in ecmwfbufrmodule.o
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _int_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buukey in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_bupkey in ecmwfbufrmodule.o
      ...
  "_PyExc_AttributeError", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _fortran_setattr in fortranobject.o
  "_PyExc_ImportError", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
  "_PyExc_MemoryError", referenced from:
      _string_from_pyobj.constprop.3 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_getcode in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_open_fortran_stdout in ecmwfbufrmodule.o
  "_PyExc_RuntimeError", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _fortran_call in fortranobject.o
  "_PyExc_TypeError", referenced from:
      _fortran_call in fortranobject.o
      _array_from_pyobj in fortranobject.o
  "_PyExc_ValueError", referenced from:
      _string_from_pyobj.constprop.3 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_getcode in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_open_fortran_stdout in ecmwfbufrmodule.o
      _array_from_pyobj in fortranobject.o
  "_PyFloat_Type", referenced from:
      _double_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_fmmh in ecmwfbufrmodule.o
  "_PyImport_ImportModule", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
  "_PyLong_AsLongLong", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbseek in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbwrite in ecmwfbufrmodule.o
  "_PyMem_Free", referenced from:
      _fortran_dealloc in fortranobject.o
      _fortran_getattr in fortranobject.o
  "_PyMem_Malloc", referenced from:
      _fortran_getattr in fortranobject.o
  "_PyModule_GetDict", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
  "_PyNumber_Float", referenced from:
      _double_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_fmmh in ecmwfbufrmodule.o
  "_PyNumber_Int", referenced from:
      _int_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buepwtc in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_get_free_unit in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_retrieve_jbpw in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_bunexs in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_get_tables in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_bustop in ecmwfbufrmodule.o
      ...
  "_PyNumber_Long", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbseek in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbwrite in ecmwfbufrmodule.o
  "_PyOS_snprintf", referenced from:
      _fortran_getattr in fortranobject.o
  "_PyObject_GetAttrString", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _int_from_pyobj in ecmwfbufrmodule.o
      _double_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbseek in ecmwfbufrmodule.o
      ...
  "_PyObject_IsTrue", referenced from:
      _f2py_rout_ecmwfbufr_mbufr_mars_filter in ecmwfbufrmodule.o
  "_PyObject_SetAttrString", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
  "_PyObject_Str", referenced from:
      _string_from_pyobj.constprop.3 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_getcode in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_open_fortran_stdout in ecmwfbufrmodule.o
  "_PySequence_Check", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _int_from_pyobj in ecmwfbufrmodule.o
      _double_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbseek in ecmwfbufrmodule.o
      ...
  "_PySequence_GetItem", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _int_from_pyobj in ecmwfbufrmodule.o
      _double_from_pyobj in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbseek in ecmwfbufrmodule.o
      ...
  "_PyString_AsString", referenced from:
      _fortran_repr in fortranobject.o
  "_PyString_ConcatAndDel", referenced from:
      _fortran_getattr in fortranobject.o
  "_PyString_FromFormat", referenced from:
      _fortran_repr in fortranobject.o
  "_PyString_FromString", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
      _fortran_repr in fortranobject.o
      _fortran_getattr in fortranobject.o
  "_PyString_FromStringAndSize", referenced from:
      _fortran_getattr in fortranobject.o
  "_PyType_IsSubtype", referenced from:
      _long_long_from_pyobj in ecmwfbufrmodule.o
      _int_from_pyobj in ecmwfbufrmodule.o
      _double_from_pyobj in ecmwfbufrmodule.o
      _string_from_pyobj.constprop.3 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes8 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_swap_bytes4 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_pbclose in ecmwfbufrmodule.o
      ...
  "_PyType_Type", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
  "_Py_BuildValue", referenced from:
      _f2py_rout_ecmwfbufr_retrieve_settings in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_close_fortran_stdout in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_big_endian in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs2 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs1 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs0 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buukey in ecmwfbufrmodule.o
      ...
  "_Py_FindMethod", referenced from:
      _fortran_getattr in fortranobject.o
  "_Py_InitModule4_64", referenced from:
      _initecmwfbufr in ecmwfbufrmodule.o
  "__PyObject_New", referenced from:
      _PyFortranObject_New in fortranobject.o
      _PyFortranObject_NewAsAttr in fortranobject.o
  "__Py_NoneStruct", referenced from:
      _f2py_rout_ecmwfbufr_buprs2 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs1 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buprs0 in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_buukey in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_bupkey in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_bucrekey in ecmwfbufrmodule.o
      _f2py_rout_ecmwfbufr_bubox in ecmwfbufrmodule.o
      ...
  "_main", referenced from:
      start in crt1.10.6.o
ld: symbol(s) not found for architecture x86_64
collect2: error: ld returned 1 exit status
error: Command "/Users/travis/miniconda/envs/_build/bin/gfortran -Wall -g -arch x86_64 f2py_build/f2py_build/src.macosx-10.6-x86_64-2.7/f2py_build/ecmwfbufrmodule.o f2py_build/f2py_build/src.macosx-10.6-x86_64-2.7/fortranobject.o f2py_build/f2py_build/src.macosx-10.6-x86_64-2.7/f2py_build/ecmwfbufr-f2pywrappers.o -L./ -L/Users/travis/miniconda/envs/_build/lib/gcc/x86_64-apple-darwin11.4.2/4.8.5 -L/Users/travis/miniconda/envs/_build/lib -lbufr -lgfortran -o ./ecmwfbufr.so" failed with exit status 1
ERROR: build of python wrapper failed
the compilation or linking stage failed
Traceback (most recent call last):
  File "setup.py", line 433, in <module>
    provides=["pybufr_ecmwf"]
  File "/Users/travis/miniconda/envs/_build/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/Users/travis/miniconda/envs/_build/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/Users/travis/miniconda/envs/_build/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/Users/travis/miniconda/envs/_build/lib/python2.7/distutils/command/install.py", line 563, in run
    self.run_command('build')
  File "/Users/travis/miniconda/envs/_build/lib/python2.7/distutils/cmd.py", line 326, in run_command
    self.distribution.run_command(command)
  File "/Users/travis/miniconda/envs/_build/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "setup.py", line 117, in run
    _build.run(self)
  File "/Users/travis/miniconda/envs/_build/lib/python2.7/distutils/command/build.py", line 127, in run
    self.run_command(cmd_name)
  File "/Users/travis/miniconda/envs/_build/lib/python2.7/distutils/cmd.py", line 326, in run_command
    self.distribution.run_command(command)
  File "/Users/travis/miniconda/envs/_build/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/Users/travis/miniconda/envs/_build/lib/python2.7/distutils/command/build_ext.py", line 339, in run
    self.build_extensions()
  File "/Users/travis/miniconda/envs/_build/lib/python2.7/distutils/command/build_ext.py", line 448, in build_extensions
    self.build_extension(ext)
  File "setup.py", line 331, in build_extension
    ibi.build()
  File "/Users/travis/miniconda/conda-bld/work/pybufr-ecmwf-0.81/build_interface.py", line 838, in build
    self.generate_python_wrapper(source_dir)
  File "/Users/travis/miniconda/conda-bld/work/pybufr-ecmwf-0.81/build_interface.py", line 1762, in generate_python_wrapper
    raise InterfaceBuildError
pybufr_ecmwf.custom_exceptions.InterfaceBuildError
Command failed: /bin/bash -x -e /Users/travis/miniconda/conda-bld/work/pybufr-ecmwf-0.81/conda_build.sh
```